spec-main/api-web-contents-spec.ts

```diff
@@ -1521,21 +1521,37 @@ describe('webContents module', () => {
 
   describe('setBackgroundThrottling()', () => {
```

Comment:
It would be better to add tests for `getBackgroundThrottling` in a new `describe`.